File: code-coaches/easy/number-of-ones/NumberOfOnes.java
import java.util.Scanner;

public class NumberOfOnes {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();

        String numBin = Integer.toBinaryString(num);
        int numOfOnesInBin = numBin.replaceAll("0", "").length();
        System.out.println(numOfOnesInBin);

        scanner.close();
    }

}

File: code-coaches/easy/number-of-ones/number_of_ones.c
#include <stdio.h>

int get_num_of_ones_in_bin(int num)
{
    int num_of_ones_in_bin = 0;
    while (num > 0)
    {
        if (num % 2 == 1)
            num_of_ones_in_bin++;

        num /= 2;
    }
    return num_of_ones_in_bin;
}

int main() {
    int num;
    scanf("%d", &num);

    printf("%d", get_num_of_ones_in_bin(num));

    return 0;
}
